### ISSUE TITLE
Bug: 500 when trying to edit image

### ISSUE THREAD

**Issue by rpiech** [WalkingPizza/strapi-plugin-placeholder#3]
I've stumbled upon an issue when trying to add alternativeText to image, when you try to save it, it will throw 500 error. Uploading works just fine, there's only a problem when you click 'Finish' button in edit of an image. I've already tried your workaround from #2  but it wouldn't work in this case.

`@strapi/strapi: "4.2.0"`
`strapi-plugin-placeholder: "^4.1.7"`

How to replicate:

1. Upload an image in media library
2. Click edit button
3. Try to save
4. It should throw 500 error

![Zrzut ekranu 2022-07-8 o 11 32 12](https://user-images.githubusercontent.com/59171898/177963082-aad85649-bed6-4e23-a6d4-978c0ce366f6.png)

**Comment by Evilazio** [comment]
I created a pacth for this error.

first install this `npm i patch-package --save-dev`

create a folder named `patches` in the root
and put a file named `strapi-plugin-placeholder+4.1.7.patch` there with this content:

```patch
diff --git a/node_modules/strapi-plugin-placeholder/server/utils.js b/node_modules/strapi-plugin-placeholder/server/utils.js
index afee787..59d836f 100644
--- a/node_modules/strapi-plugin-placeholder/server/utils.js
+++ b/node_modules/strapi-plugin-placeholder/server/utils.js
@@ -8,7 +8,7 @@ const pluginId = require('./plugin-id');
  * @returns {boolean} whether a placeholder can be generated for the given MIME type
  */
 
-const isValidMimeType = (mimeType) => mimeType.startsWith('image/');
+const isValidMimeType = (mimeType) => mimeType && mimeType.startsWith('image/');
 
 /**
  * Helper that retrieves one of the available services of this plugin from Strapi.
```

after this rerun `npm i`

**Comment by WalkingPizza** [comment]
@Evilazio can you confirm the placeholder is updated properly when you update a media library entry by replacing the image in it?

**Comment by arixwap** [comment]
I also face the same issue here

![image](https://user-images.githubusercontent.com/11769214/186836538-48687659-1611-4a73-a937-c35ae39f23cf.png)

**Comment by Evilazio** [comment]
> @Evilazio can you confirm the placeholder is updated properly when you update a media library entry by replacing the image in it?

Yes, the placeholder is updated correctly when replace the image.

**Comment by WalkingPizza** [comment]
Should be fixed in the new 4.3.6 version!
